src/helpers/components/Three.js

```diff
@@ -0,0 +1,8 @@
+'use client'
+
+import { r3f } from '@/helpers/global'
+
+export const Three = ({ children }) => {
+  // @ts-ignore
+  return <r3f.In>{children}</r3f.In>
+}
```

Comment:
ran this with `-ts` and this component errors out because it becomes `Three.ts`

Also removed the `// @ts-ignore` and ran into no errors here. possibly unneeded?

I like the `// @ts-expect-error` directive for use cases like this because it will throw an error if the below error gets resolved